### PR TITLE
update data request

### PR DESCRIPTION
**Pull request by DanielJMaher**

### CODE REVIEW

File: ooiservices/app/uframe/controller.py [hunk 1]
```diff
@@ -492,13 +492,13 @@ def get_uframe_plot_contents_chunked(mooring, platform, instrument, stream_type,
     '''
     try:
         if dpa_flag == '0' and len(parameter_ids)<1:
-            query = '?beginDT=%s&endDT=%s&limit=5000' % (start_time, end_time)
+            query = '?beginDT=%s&endDT=%s&limit=%s' % (start_time, end_time, current_app.config['DATA_POINTS'])
         elif dpa_flag == '1' and len(parameter_ids)<1:
-            query = '?beginDT=%s&endDT=%s&limit=5000&execDPA=true' % (start_time, end_time)
+            query = '?beginDT=%s&endDT=%s&limit=%s&execDPA=true' % (start_time, end_time, current_app.config['DATA_POINTS'])
```
Comment (birdage) [inline ooiservices/app/uframe/controller.py]: @DanielJMaher do we need exec dpa anymore?

Comment (DanielJMaher) [inline ooiservices/app/uframe/controller.py]: @birdage No

from https://uframe-cm.ooi.rutgers.edu/projects/ooi/wiki/Web_Interface 

execDPA -- deprecated. Used on the uFrame/EDEX backend to generate derived products. Not used by the stream_engine backend.

**Comment by DanielJMaher** [comment]
@birdage?  Merge?

**Comment by DanielJMaher** [comment]
I have Bird's Permission

**Pull request merged**